VerilogCodingStyle.md

```diff
@@ -2163,10 +2186,11 @@ always_comb begin
 end
 ```
 
-##### Wildcards in case items
+#### Wildcards in case items
 
-Use `case` instead of `casez` whenever wildcard operator behavior is not
-required. When wildcard behavior is needed, use `casez`.
+Use `case` instead of `case inside` nor `casez` whenever wildcard operator
+behavior is not required. When wildcard behavior is needed, use `case inside`
+(or `casez` if only Verilog-2001 is supported).
```

Comment:
```suggestion
Use `case` if wildcard operator behavior is not needed. 
Use `case inside` if wildcard operator behavior is needed.
Use `casez` if wildcard operator behavior is needed and Verilog-2001 compatibility is required.
```

(The last statement slightly contradicts this style guide, which requires SystemVerilog, but I'd still keep the `casez` guidance in here for cases where we need to please a random tool.)

Please also update the example further up in the `Case Statements` section, where `casez` is currently used.